packages/model/DimensionMetadata.ts

```diff
@@ -19,6 +20,9 @@ function DimensionMetadataMixin<Base extends Constructor>(Resource: Base) {
   class Impl extends Resource implements DimensionMetadata {
     @property()
     about!: NamedNode
+
+    @property({ values: 'array' })
+    name?: Literal[]
```

Comment:
This will never be undefined. No triples = empty array
```suggestion
    name: Literal[]
```